scalableminds/webknossos
# Save expanded/collapsed state of segment groups

### Steps to test:
- Open an annotation and go to segments tab. Create a group hierarchy (around three levels min) with some segments in each group.
- Expand and collapse some groups, remember state.
- Change to another tab and back. The state (groups expanded/collapsed) should be the same.
- Reload the page. The state should still be the same 
- Test that old behavior is still working. Expanding and collapsing via arrow icon, expansion/collapse of all subgroups etc.

### TODOs:
- [x] fix bug that expand children on root doesnt work after collapsing all subgroups

### Issues:
- fixes #7825 

------
(Please delete unneeded items, merge only when none are left open)
